### PR TITLE
Bugfix 3.10/stop total pregel time correctly

### PR DESCRIPTION
**Pull request by jvolmer**
### Scope & Purpose

If Pregel config uses store: false, the total timer now stops directly after the computation. (before, it stopped only when the run was canceled, which gave in most cases a much too large runtime in the Pregel status).

Additionally, this PR corrects memory label in status output.

- [x] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*
  - [ ] Backport for 3.7: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: arangod/Pregel/Status/Status.h [hunk 1]
```diff
@@ -115,7 +115,7 @@ template<typename Inspector>
 auto inspect(Inspector& f, GraphStoreStatus& x) {
   return f.object(x).fields(f.field("verticesLoaded", x.verticesLoaded),
                             f.field("edgesLoaded", x.edgesLoaded),
-                            f.field("memoryUsed", x.memoryBytesUsed),
+                            f.field("memoryBytesUsed", x.memoryBytesUsed),
```
Comment (jsteemann) [inline arangod/Pregel/Status/Status.h:118]: Are there any users of this attribute that would need to be adjusted as well after the name change?

**Review by jsteemann — APPROVED** [review]
LGTM, just left a question about an attribute name change.

File: arangod/Pregel/Status/Status.h (hunk 1 above)
Comment (jvolmer) [reply arangod/Pregel/Status/Status.h:118]: No, because the field was just introduced 3 weeks ago: https://github.com/arangodb/arangodb/pull/16515

**Review by KVS85 — APPROVED** [review]
LGTM.

**Pull request merged**